src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -11,23 +14,28 @@ export interface LinkButtonProps
     HTMLAnchorElement
   > {
   href: string;
+  isBlank: boolean;
```

Comment:
I can't find a place with `<LinkButton isBlank />`

Reply:
it's right here https://github.com/sparkletown/sparkle/blob/d592e3dfb39e1927aca1965a5d324915ee8c3de9/src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx